amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -15,6 +15,12 @@
       <app.amp.client.war.folder>${project.build.directory}/${project.build.finalName}-war</app.amp.client.war.folder>
 
       <image.name>alfresco/alfresco-governance-repository-community-base</image.name>
+
+      <local.registry.host>127.0.0.1</local.registry.host>
+      <local.registry.port>5000</local.registry.port>
+      <base.image>alfresco/alfresco-community-repo-base</base.image>
+      <base.image.tag>latest</base.image.tag>
```

Comment:
Why is this hard-coded to `latest`? What about the scenarios where `image.tag` is pointing to a specific version rather than latest?

Reply:
Right! I replaced it with `${image.tag}`.